packages/components/src/drop-zone/animators/index.ts

```diff
@@ -0,0 +1 @@
+export { MaybeFade } from './maybe-fade';
```

Comment:
I named this 'animator' because it's really more of an orchestrator than what defines the animation per se. The animation is defined in the CSS. The extraction is to make it easier to further abstract it and share between components later if needed.